src/components/Common.js

```diff
@@ -9,14 +9,14 @@ const screenBreakpoints = {
   desktop: 1200,
 }
 
-export const maxWidthMediaQueries = ((size) => {
+export const maxWidthMediaQueries = (size) => {
   return `@media only screen and (max-width: ${screenBreakpoints[size]}px)`
-})
+}
 
 export const CardLike = css`
   padding: 2em;
   border-radius: 3px;
-  background-color: ${p => p.theme.colors.secondaryBackground};
+  background-color: ${(p) => p.theme.colors.secondaryBackground};
```

Comment:
How do we turn this rule off?

Reply:
I think we can add `arrowParens: "avoid"`, let me try that